src/components/StructuredNavigation/NavUtils/SectionHeading.js

```diff
@@ -30,12 +30,13 @@ const SectionHeading = ({
   sectionRef,
   structureContainerRef,
 }) => {
-  // Always collapse root structure element
-  const [isOpen, setIsOpen] = useState(isRoot);
+  const { isCollapsed } = useCollapseExpandAll();
+  // root structure items are always expanded
+  const [sectionIsCollapsed, setSectionIsCollapsed] = useState(
+    isRoot ? false : true || isCollapsed);
```

Comment:
Is this always going to be `true` for non-root structure items?

Reply:
This sets the value on initial render, and then changed as user interacts with the structure and the collapse/expand all sections button.

Reply:
I think I'm confused about the boolean logic and what it trying to be achieved with `|| isCollapsed`.  If I'm reading it correctly `isCollapsed` isn't being used at all because `true` will always be returned when `isRoot == false`.

Reply:
Correct! Sorry, I was mistaking this with a previous iteration of the implementation I had in mind. I'll fix it. Thanks!